nginx/nginx_test.go

```diff
@@ -2044,6 +2044,12 @@ func TestUpdatesMetricsFromNginxStatusPage(t *testing.T) {
 	assertIngressRequestCounters(t,
 		"heapster.sandbox.cosmic.sky", "/",
 		2012.0, 1099.0, 0.0, 7.0, 0.0, 0.0, 0.0)
+	assertIngressRequestCounters(t,
+		"duplicate-path.sandbox.cosmic.sky", "/path/",
+		5000.0, 2000.0, 0.0, 5.0, 0.0, 0.0, 0.0)
+	assertIngressRequestCounters(t,
+		"misconfigured-ingress.sandbox.cosmic.sky", "/bad/",
+		0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0)
```

Comment:
it is not really clear to me what is the difference between these two.
Is it to prove that a `bad path with a valid duplicate` will report the valid count, whereas a `bad path without a valid duplicate'  will report 0?

Reply:
Correct. I've changed the names of these hosts and added comments that will hopefully express the tests intentions